load_data.sh

```diff
@@ -31,4 +30,8 @@ bazel-bin/tf_main --filename efficientnet_b5_feature-vector_1_ImageFeatureVector
 bazel-bin/tf_main --filename progan-128_1_ImageGenerator_SavedModel.pb --model_name progan-128_1 --category ImageGenerator --is_saved_model True
 bazel-bin/tf_main --filename spiral_default-fluid-gansn-celebahq64-gen-19steps_1_ImageRNNAgent_SavedModel.pb --model_name spiral_default-fluid-gansn-celebahq64-gen-19steps_1 --category ImageRNNAgent --is_saved_model True
 bazel-bin/tf_main --filename unet_industrial_class_1_1_ImageSegmentation_SavedModel.pb --model_name unet_industrial_class_1_1 --category ImageSegmentation --is_saved_model True
-bazel-bin/tf_main --filename vision_embedder_inaturalist_V2_1_ImageFeatureVector_SavedModel.pb --model_name vision_embedder_inaturalist_V2_1 --category ImageFeatureVector --is_saved_model True
\ No newline at end of file
+bazel-bin/tf_main --filename vision_embedder_inaturalist_V2_1_ImageFeatureVector_SavedModel.pb --model_name vision_embedder_inaturalist_V2_1 --category ImageFeatureVector --is_saved_model True
+bazel-bin/tf_main --filename tweening_conv3d_bair_1_VideoGeneration_SavedModel.pb --model_name tweening_conv3d_bair_1 --category VideoGeneration --is_saved_model True 
```

Comment:
is_saved_model default value can be 'True' as most of them are saved models.

Reply:
Fixed